recipes/libxml2/all/conanfile.py

```diff
@@ -255,7 +255,7 @@ def package_info(self):
             self.output.info("Appending PATH environment variable: {}".format(bindir))
             self.env_info.PATH.append(bindir)
 
-        if self.settings.os == "Linux" or self.settings.os == "Macos":
+        if self.settings.os in ["Linux", "Macos", "FreeBSD"]:
```

Comment:
Maybe just put it in `else` block for a `if self.settings.os == "Windows"`?

Reply:
I"m not sure, but `m` can also be used with `mingw`.
So testing for Windows is not enough.

Reply:
That's correct. Mingw makes life harder.

Reply:
I'm not sure either, it's usually better to be explicit in this case